### PR TITLE
Added nonRegistedUser.spec.ts and teacherCanLoginAndFindStudent.spec.…

### PR DESCRIPTION
**Pull request by Kristinochka-bee**
Added nonRegistedUser.spec.ts and teacherCanLoginAndFindStudent.spec.ts and described pages necessary for these tests

### CODE REVIEW

File: pages/StudentDetailsPage.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+export class StudentDetailsPage extends PageObject{
+    protected readonly studentName: Locator;
+
+    constructor(page:Page) {
+        super(page,'/');
+        this.studentName = page.locator('//*[@data-field-type="heading1"]');
+    }
+
+    async checkUserOnCorrectStudentDetailsPage(name:string){
+        //await expect(this.studentName);
```
Comment (LeonadO555) [inline pages/StudentDetailsPage.ts]: Need to remove comment code

File: pages/HomePage.ts [hunk 2]
```diff
@@ -23,6 +25,11 @@ export class HomePage extends PageObject {
         this.studentDirectoryButton = page.locator('//span[normalize-space()=\'Student Directory\']');
         this.avatarButton = page.locator('//span[normalize-space()=\'Student Directory\']');
         this.singOutButton = page.locator('//span[normalize-space()=\'Sign Out\']');
+        this.coursesButton = page.locator('//span[normalize-space()=\'Courses\']');
+        this.courseList = page.locator('//*[@class=\'list-container\']');
```
Comment (LeonadO555) [inline pages/HomePage.ts:29]: duplicate

File: pages/StudentDetailsPage.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        const textFromLocator = await this.studentName.textContent();
+        return textFromLocator.includes(name);
+    }
+
+    async check(name:string){
```
Comment (LeonadO555) [inline pages/StudentDetailsPage.ts]: What is check??
I don't understand it
maybe 
checkCountainStudentName?

**Review by LeonadO555 — APPROVED** [review]

File: pages/HomePage.ts (hunk 2 above)
Comment (Kristinochka-bee) [reply pages/HomePage.ts:29]: fixed

File: pages/StudentDetailsPage.ts (hunk 1 above)
Comment (Kristinochka-bee) [reply pages/StudentDetailsPage.ts]: removed

File: pages/StudentDetailsPage.ts (hunk 3 above)
Comment (Kristinochka-bee) [reply pages/StudentDetailsPage.ts]: fixed

**Review by LeonadO555 — APPROVED** [review]

**Pull request merged**